### ISSUE TITLE
Manage user groups independently for user creation

### ISSUE THREAD

**Issue by dominique-vassard** [couchbase-partners/helm-charts#42]
Hello,
today a group is created for each user defined in the  yaml, which is not ideal for mass user creation an role bindings.
See  helm-charts/couchbase-operator/templates/couchbase-users.yaml whcih creates a group for each user.

Thank you.

**Comment by tahmmee** [comment]
Agree.  This is being considered for the 2.2.x supported release of the chart.
